src/Entity/Notification.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\NotificationRepository;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: NotificationRepository::class)]
+class Notification
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\ManyToOne(inversedBy: 'notifications')]
+    #[ORM\JoinColumn(nullable: false)]
+    private ?NotificationDefinition $definitionId = null;
+
+    #[ORM\Column]
+    private ?bool $status = null;
+
+    #[ORM\Column]
+    private ?\DateTimeImmutable $createdAt = null;
```

Comment:
Wykorzystaj do tego Timestampable i to updatedAt też.

https://symfony.com/doc/6.3/doctrine.html#doctrine-extensions-timestampable-translatable-etc